tests/integration/test_charm.py

```diff
@@ -395,21 +393,57 @@ async def test_promote_user(
     act: when promoting a user
     assert: the user should be promoted successfully.
     """
+    with requests.session() as sess:
+        res = sess.get(f"{discourse_address}/session/csrf", headers={"Accept": "application/json"})
```

Comment:
s/res/response would increase readability

Reply:
have done so, and have made consistent across the conftest also.

Reply:
This has been resolved.

Reply:
(and has been applied elsewhere, where relevant.)